testcases/kselftests.yaml

```diff
@@ -1,2 +1,4 @@
 {% extends "testcases/master/template-kselftest.yaml.jinja2" %}
 
+{% set testnames = [''] %}
```

Comment:
Shall we stop long running full job ?
testcases/kselftests.yaml 
The timeout is 85 min.

Reply:
long running full jobs yes we can handle that from the variables file.